libcxx/test/std/ranges/range.adaptors/range.stride.view/ctad.compile.pass.cpp

```diff
@@ -0,0 +1,37 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03, c++11, c++14, c++17, c++20
+
+// template <class _Range>
+// stride_view(_Range&&, range_difference_t<_Range>) -> stride_view<views::all_t<_Range>>;
```

Comment:
The CTAD test should not be a compile-only test -- it should also actually run the code where we perform deduction.

Reply:
Got it! Thanks!!

Reply:
I believe that this is addressed in aa7c07b6f6b9aff728a9870b91f9f3ce31f825c2.